.github/workflows/bazel_build_integrations.yml

```diff
@@ -23,31 +23,31 @@ on:
 jobs:
   linux:
     runs-on: ubuntu-18.04
+    container: l.gcr.io/google/bazel:2.1.0
     env:
       CXX: clang++
       CC: clang
       PYTHON_BIN: /usr/bin/python3
-      # Install the version of Bazel that is required by our .bazelversion file.
-      BAZEL_VERSION: 2.1.0
     steps:
-      - name: Installing bazel
+      - name: Installing IREE dependencies
         run: |
-          # https://docs.bazel.build/versions/master/install-ubuntu.html
-          sudo apt-get install unzip zip
-          wget https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION?}/bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          chmod +x bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          ./bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh --user
-          rm bazel-${BAZEL_VERSION?}-installer-linux-x86_64.sh
-          echo "::add-path::${HOME?}/bin"
-      - name: Installing dependencies
+          apt-get update
+          apt-get install -y clang libsdl2-dev
+          # Update pip before using it so it picks up the latest versions of our
+          # dependencies.
+          apt-get install -y python3 python3-pip python3-setuptools
+          python3 -m pip install --upgrade pip
+          python3 -m pip install numpy tf-nightly
+      - name: Installing actions/checkout@v2 dependencies
+        run: |
+          # Update git to version >= 2.18 for actions/checkout@v2.
+          add-apt-repository ppa:git-core/ppa
+          apt-get update
+          apt-get install -y git
+      - name: Installing tensorflow dependencies
```

Comment:
Maybe move tf-nightly to the tensorflow dependencies section?

Reply:
I was debating with myself about that. I left them separate because here it seems that tensorflow is acting as a dependency of IREE, where as python2 numpy is a dependency of tensorflow that we hope to eventually remove. I could move tensorflow into this section and rename it to "Installing tensorflow and its dependencies", it just wont match the blocks in the other files. What would you prefer?

Reply:
:shrug: fair enough, this is fine